spec/unit/rpm_database_spec.rb

```diff
@@ -0,0 +1,215 @@
+# Copyright (c) 2013-2015 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of version 3 of the GNU General Public License as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.   See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE about this file by physical or electronic mail,
+# you may find current contact information at www.suse.com
+
+require_relative "spec_helper"
+
+describe RpmDatabase do
+  describe RpmDatabase::ChangedFile do
+    describe "config_file?" do
+      it "returns true for config files" do
+        file = RpmDatabase::ChangedFile.new("c",
+          name:    "/etc/foo",
+          status:  "changed",
+          changes: ["md5"]
+        )
+        expect(file.config_file?).to be(true)
+      end
+
+      it "returns false for non-config files" do
+        file = RpmDatabase::ChangedFile.new("",
+          name:    "/etc/foo",
+          status:  "changed",
+          changes: ["md5"]
+        )
+        file.type = ""
+        expect(file.config_file?).to be(false)
+      end
+    end
+  end
+
+  let(:system) { LocalSystem.new }
+  let(:changed_files_sh_result) { File.read(File.join(Machinery::ROOT,
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.